changelogs/head.asciidoc

```diff
@@ -22,3 +22,4 @@ https://github.com/elastic/apm-server/compare/8.15\...main[View commits]
 ==== Added
 
 - APM Server will no longer retry an HTTP request that returned 502s, 503s, 504s. It will only retry 429s. {pull}13523[13523]
+- Enable self-instrumentation tracing again with transaction sampling support {pull}14231[14231]
```

Comment:
I think we need to reword the changelog entry to make it actionable, i.e. what does this mean for users? How do they benefit from this change? @axw @simitt how do you envision users will use this and how could we phrase it nicely so it's not cryptic?

Reply:
Hm, well I think ideally the functionality would be documented at https://www.elastic.co/guide/en/fleet/master/elastic-agent-monitoring-configuration.html, and we'd just mention here that we have enabled support for APM instrumentation of APM Server itself when running under Elastic Agent.

In the mean time... I don't know. I think what's written is OK, but yes a little bit cryptic. What about:

> APM Server now supports emitting distributed tracing for its own operation when running under Elastic Agent, and adds support for configuring a sampling rate